### PR TITLE
Parse Group test data from TSV file

### PR DESCRIPTION
**Pull request by himaniyadav**
Load test data from a TSV file into the database.

- ParseGroupDataServlet is called by the webapp's web.xml file. 
- Set load-on-startup value to positive when you need to populate the datastore with test group data (for example, after a mvn-clean). 
- Once populated, set back to a negative number to avoid duplicate data.

Here is what one of the test group pages looks like:
image

**Note**: Members section is blank because even though each group has a list of memberIds, those Users don't exist in my local Datastore. Once Users are loaded in it should display properly, but I can still work on my feature using the group data regardless.

### CODE REVIEW

File: capstone/src/main/java/com/google/sps/servlets/ParseGroupDataServlet.java [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+import com.google.sps.Objects.Group;
+import com.google.sps.Objects.response.PostResponse;
+import com.google.sps.servlets.ServletHelper;
+import error.ErrorHandler;
+
+/**
+ * Parses a TSV file of fake group data and populates the Datastore with Group and Post entities.
+ */
+@WebServlet("/parse-group-data")
+public class ParseGroupDataServlet extends HttpServlet {
```
Comment (g-asweet) [inline capstone/src/main/java/com/google/sps/servlets/ParseGroupDataServlet.java]: `ParseTestGroupDataServlet` and anything else regarding fake data should be obnoxiously pointing out this is for testing only

File: capstone/src/main/webapp/WEB-INF/group-test-data.tsv [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+Avatar	ISABELA_JACKSON, ISABELLA_MARTIN, ISABELLE_LEE, ISIS_PEREZ, ITZEL_THOMPSON, IZABELLA_WHITE	Okay wow the ending to Book 1 was intense! I love Zuko's character development so far and am excited to see more of his sister	Wow you finished this week's episodes quickly! I'm not gonna spoil anything but you will love Book 2! Lots of fun Zuko and Azula times ahead	I was kind of shook when (spoilers) that girl turned into the moon. And then Aang turned into a fish monster? Spirits are confusing	Lol the moon, her name is Yue okay? And she will be respected! Anyway I agree that you guys are not ready for Book 2, it's gonna be amazing! Glad we finished Book 1 at last	This week's episodes were so fun in my opinion! (Spoilers) I am sad that Yue sacrificed herself because poor Sokka deserves happiness!
```
Comment (g-asweet) [inline capstone/src/main/webapp/WEB-INF/group-test-data.tsv:5]: https://i.imgur.com/zEMF6eS.gif

File: capstone/src/main/webapp/WEB-INF/group-test-data.tsv [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+Music Makers	JAYLA_WRIGHT, JAYLAH_SCOTT, JAYLEEN_TORRES	Viola gang rise up! Jk I know we'll never be respected. Anyway, here is my attempt at Shostakovich's viola sonata	I didn't know Shostakovich wrote a viola sonata, that seems...sacreligious. Lol just kidding. I decided to mix it up so instead of classical, here's a fun jazzy piano piece	You can't say a viola sonata is "sacreligious" when you branch out from classical! Smh! Also I'll have you know that Brahms and many others also wrote a viola piece for us!!	I'm trying to learn guitar but I was too lazy to tune it this morning so here's a piece on the recorder instead, hope you like it	While idk if this can ever live up to the recorder,  (simply amazing lol) here's my take on a classical piece
```
Comment (g-asweet) [inline capstone/src/main/webapp/WEB-INF/group-test-data.tsv:7]: https://pics.me.me/anyway-heres-wonderwall-11823057.png

File: capstone/src/main/webapp/WEB-INF/web.xml [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://xmlns.jcp.org/xml/ns/javaee
+         http://xmlns.jcp.org/xml/ns/javaee/web-app_3_1.xsd"
+         version="3.1">
+  <servlet>
+    <servlet-name>parse-group-data</servlet-name>
+    <servlet-class>com.google.sps.servlets.ParseGroupDataServlet</servlet-class>
+    <!--Set value to positive if you need to populate the datastore with test group data.
+        Once populated, set back to a negative number to avoid duplicate data. -->
+    <load-on-startup>-1</load-on-startup>
```
Comment (g-asweet) [inline capstone/src/main/webapp/WEB-INF/web.xml:12]: Neat!

**Review by g-asweet — APPROVED** [review]

File: capstone/src/main/webapp/WEB-INF/group-test-data.tsv (hunk 2 above)
Comment (himaniyadav) [reply capstone/src/main/webapp/WEB-INF/group-test-data.tsv:5]: https://i.imgur.com/X5tD7M5.gif

**Pull request merged**